postgres/templates/secrets.yml

```diff
@@ -0,0 +1,12 @@
+{{ if not .Values.vault_secrets.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "postgres.fullname" . }}-postgres
+  type: Opaque
+  stringData:
+    {{- with .Values.jupyterhub_db }}
+    jupyterhub_password: '{{ .Values.jupyterhub_db.password}}'
```

Comment:
Wouldn't it make more sense to store this in Vault instead of having the chart define a regular `Secret` and expect the secret to be in the `values.yaml`?

Reply:
These two were cribbed from ... uh ... some other chart, but basically it's a switch to say "if we're not using Vault Secrets Operator then we will be supplying a regular secret."

We should always be using Vault Secrets Operator, of course.